src/sql/workbench/parts/query/browser/modelViewTab/media/dialogModal.css

```diff
@@ -33,8 +27,8 @@
 	flex-direction: column;
 	width: 100%;
 	height: 100%;
-	overflow-x: hidden;
-	overflow-y: scroll;
+	overflow-x: auto;
+	overflow-y: auto;
```

Comment:
What if the content size is large enough that auto overflows?  Is this contained within a scrollable view?

Reply:
if the content doesn't fit in the available space, this div will be scrolled.